bundles/AdminBundle/Resources/public/js/pimcore/helpers.js

```diff
@@ -18,6 +18,22 @@ pimcore.helpers.sanitizeEmail = function (email) {
     return email.replace(/[^a-zA-Z0-9_\-@.+]/g,'');
 };
 
+
+// Note: Only run this once. And don't run it on already encoded strings e.g. &amp; becomes &amp;amp;
+pimcore.helpers.escapeHtml = function (html) {
```

Comment:
using `Ext.util.Format.htmlEncode()`  should do the trick as well.

Reply:
Thanks for the hint. `Ext.String.hasHtmlCharacters` is even more useful in this case to check if Html characters are present.